sdf/opIntersection.glsl

```diff
@@ -11,6 +11,15 @@ use: <float> opIntersection( in <float> d1, in <float> d2 [, <float> smooth_fact
 
 float opIntersection( float d1, float d2 ) { return max(d1,d2); }
 
+Material opIntersection(Material d1, Material d2)
```

Comment:
Maybe add:

#if defined(STR_MATERIAL)

in case people use this function without material definition?
or
add #include "material.glsl" at the top